src/app/timed-call/timed-call.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'timed-call',
+  templateUrl: './timed-call.component.html',
+  styleUrls: ['./timed-call.component.scss']
+})
+export class TimedCallComponent implements OnInit, OnDestroy {
+
+  // Function that will be called everytime the Timer reaches 0.
+  @Input()
+  functionToCall: Function;
+
+  // Optional parameter to define the interval times that are needed in the parent Component. This overwrites the default of 10, 30, 60, 120 and 300 seconds if provided
+  @Input()
+  delayOptions?: number[];
+
+  // Optional parameter to provide another label for the Timer Component than "Refresh Rate"
+  @Input()
+  label?: string;
+
+  public callInterval: any;
+  public currentCallDelay: number;
+  public defaultDelayOptions: number[] = [10, 30, 60, 120, 300];
+  public circleVisible: boolean = true; 
+
+  constructor() { }  
+
+
+  ngOnInit(): void {
+    // Establish timeout interval, get the named reference for it to update it later
+    this.label = this.label ? this.label : "Refresh Rate"
```

Comment:
I'm not entirely sure, but I think you should be able to specify the default value right at the point where you declare the label property: `@Input() label = 'Refresh Rate';`